Library/Homebrew/test/test_formula_installer_bottle.rb

```diff
@@ -0,0 +1,50 @@
+require "testing_env"
+require "formula"
+require "compat/formula_specialties"
+require "formula_installer"
+require "keg"
+require "testball_bottle"
+
+class InstallBottleTests < Homebrew::TestCase
+	module OS
```

Comment:
Instead of manually stubbing this, you can use the [mocha library](https://github.com/freerange/mocha), which is already used as part of the tests. Something ike this should work:

```ruby
def test_something
  MacOS.expects(:has_apple_developer_tools?).returns(false)
  # rest of your test goes here
end
```